python/pyrogue/_Variable.py

```diff
@@ -447,6 +449,10 @@ def verify(self):
     def base(self):
         return self._base
 
+    def _setDefault(self):
```

Comment:
This is to avoid the device functions called with normal setDisp?

Reply:
I need to go back and trace why it is, but my recollection is that the `defaults` passed to a Device constructor don't work unless this is here.

Reply:
I'm still trying to trace this, but I think it's that `Block._setDefault()` needs to be called to properly stage defaults for RemoteVariables. `BaseVariable.setDefault()` will just call `self.setDisp(write=False)`, which doesn't quite do the right thing.

It is possible though that some underlying `Block.set()` implementation has since changed and that this is no longer necessary.